python_apps/pypo/pypo/pypofile.py

```diff
@@ -88,8 +88,8 @@ def copy_file(self, media_item):
                 os.chmod(dst, stat.S_IRUSR | stat.S_IWUSR | stat.S_IRGRP | stat.S_IROTH)
 
                 if media_item["filesize"] == 0:
-                    file_size = self.report_file_size_and_md5_to_airtime(
-                        dst, media_item["id"], host, username
+                    file_size = self.report_file_size_and_md5_to_libretime(
```

Comment:
While you are renaming the function, could we replace `libretime` with `api`, it is explicit and I think we shouldn't use the project name anyway.